gems/aws-sdk-core/lib/aws-sdk-core/plugins/user_agent.rb

```diff
@@ -25,12 +29,20 @@ class UserAgent < Seahorse::Client::Plugin
 
       def self.feature(feature, &block)
         Thread.current[:aws_sdk_core_user_agent_feature] ||= []
-        Thread.current[:aws_sdk_core_user_agent_feature] << "ft/#{feature}"
+        Thread.current[:aws_sdk_core_user_agent_feature] << feature
```

Comment:
I can't remember if we discussed this in the previous UA implementation - but does this cause a problem for things like S3 Access Grants where multiple SDK requests are made in the same thread or does the block with pop at the end fully handle that?

Reply:
I don't believe so. I think that was the case until my last commit which made it block based. You should confirm that. But in express session auth testing, both credentials call (create session) and get object call will have these user agents:
```
user agent: aws-sdk-ruby3/3.196.1 ua/2.1 api/s3#1.151.0 os/macos#23 md/x86_64 lang/ruby#3.3.1 md/3.3.1 m/A
user agent: aws-sdk-ruby3/3.196.1 ua/2.1 api/s3#1.151.0 os/macos#23 md/x86_64 lang/ruby#3.3.1 md/3.3.1 m/A
```

Reply:
I've changed this so that create session does not have the metric. I'm getting clarification.